docs/components/ComponentProps.tsx

```diff
@@ -0,0 +1,231 @@
+        {ref && (
+          <TableRow>
+            <TableCell>Ref</TableCell>
+            <TableCell>{ref}</TableCell>
+            <TableCell></TableCell>
+            <TableCell>
+              A ref pointed to the element rendered by this component.
+            </TableCell>
+          </TableRow>
+        )}
+      </TableBody>
+    </Table>
+  );
+}
+
+function DescriptionListProps({
+  children,
+  props,
+  ref,
+  collapsed
+}: Props & { collapsed?: boolean }) {
+  return (
+    <ul className="Component__Props">
+      {children && (
+        <li>
+          <DescriptionList collapsed={collapsed}>
+            <DescriptionListItem>
+              <DescriptionTerm>Name</DescriptionTerm>
+              <DescriptionDetails>
+                children{' '}
+                {(children as Record<string, unknown>)?.required && (
+                  <Tag>Required</Tag>
+                )}
+              </DescriptionDetails>
+            </DescriptionListItem>
+            <DescriptionListItem>
+              <DescriptionTerm>Type</DescriptionTerm>
+              <DescriptionDetails>
+                {children === true || !children?.type
+                  ? 'React.ReactNode'
+                  : Array.isArray(children.type)
+                  ? children.type.join('| ')
+                  : children.type}
+              </DescriptionDetails>
+            </DescriptionListItem>
+            <DescriptionListItem>
+              <DescriptionTerm>Description</DescriptionTerm>
+              <DescriptionDetails>Child content.</DescriptionDetails>
+            </DescriptionListItem>
+          </DescriptionList>
+        </li>
+      )}
+      {props.map(
+        (
+          { name, type, required, deprecated, defaultValue, description },
+          index
+        ) => (
+          <li key={name}>
+            <DescriptionList collapsed={collapsed}>
+              <DescriptionListItem>
+                <DescriptionTerm>Name</DescriptionTerm>
+                <DescriptionDetails>
+                  {name} {required && <Tag>Required</Tag>}{' '}
+                  {deprecated && <Tag>deprecated</Tag>}
+                </DescriptionDetails>
+              </DescriptionListItem>
+              <DescriptionListItem>
+                <DescriptionTerm>Type</DescriptionTerm>
+                <DescriptionDetails>
+                  {Array.isArray(type) ? type.join(' | ') : type}
+                </DescriptionDetails>
+              </DescriptionListItem>
+              <DescriptionListItem>
+                <DescriptionTerm>Default</DescriptionTerm>
+                <DescriptionDetails>{defaultValue}</DescriptionDetails>
+              </DescriptionListItem>
+              <DescriptionListItem>
+                <DescriptionTerm>Description</DescriptionTerm>
+                <DescriptionDetails>{description}</DescriptionDetails>
+              </DescriptionListItem>
+            </DescriptionList>
+          </li>
+        )
+      )}
+      {children && (
```

Comment:
I'm guessing this is supposed to be:
```suggestion
      {ref && (
```

Reply:
Yup, good catch!